Bottom: 8aa807451bb5ebdb5430f04ac877aee91ba0d96a
Top:    9b1dafcb79431e0d4a5e4b7c0df3c9b42b6a6a61
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-09-30 19:54:29 +0100

Draw pad bounding box to check its size


---

diff --git a/src/draw.c b/src/draw.c
index f710567..38daf9a 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1523,6 +1523,16 @@ DrawPadLowLevel (hidGC gc, PadTypePtr Pad, Boolean clear, Boolean mask)
                       Pad->Point1.X, Pad->Point1.Y,
                       Pad->Point2.X, Pad->Point2.Y);
     }
+#if 0
+  { /* Draw bounding box for test */
+    BoxType *box = &Pad->BoundingBox;
+    gui->set_line_width (gc, 1);
+    gui->draw_line (gc, box->X1, box->Y1, box->X1, box->Y2);
+    gui->draw_line (gc, box->X1, box->Y2, box->X2, box->Y2);
+    gui->draw_line (gc, box->X2, box->Y2, box->X2, box->Y1);
+    gui->draw_line (gc, box->X2, box->Y1, box->X1, box->Y1);
+  }
+#endif
 }
 
 /* ---------------------------------------------------------------------------
